packages/at_client/lib/src/converters/encryption/aes_converter.dart

```diff
@@ -111,7 +111,8 @@ class AESDecryptionSink extends ChunkedConversionSink<List<int>> {
 
 IV getIV(String? ivBase64) {
   if (ivBase64 == null) {
-    return IV.fromLength(16);
+    // From the bad old days when we weren't setting IVs
+    return IV(Uint8List(16));
```

Comment:
This change is a defensive measure to prevent the next release of the `encrypt` package from causing us problems, as the behaviour of `IV.fromLength` has changed